pkg/result/processors/fixer.go

```diff
@@ -136,7 +136,7 @@ func (f Fixer) mergeLineIssues(lineNum int, lineIssues []result.Issue, origFileL
 		r := i.Replacement
 		if r.Inline == nil || len(r.NewLines) != 0 || r.NeedOnlyDelete {
 			f.log.Infof("Line %d has multiple issues but at least one of them isn't inline: %#v", lineNum, lineIssues)
-			return &lineIssues[0]
+			return i
```

Comment:
Do we need a unit test to cover that case?

Reply:
done

Reply:
one test enough or need more test cases?